Shopify/cli
# Enable Plus Sandbox stores

### WHY are these changes introduced?

The CLI only allows to work with development stores, but there's no reason we can also allow Shopify Plus sandbox stores.

### WHAT is this pull request doing?

- Retrieve Plus sandbox stores as well as development stores from Partners
- Update copies

### How to test your changes?

- `spin up partners`
- Create a Shopify Plus Sandbox store in Spin and restart Partners
- `bin/spin p shopify app dev --path your-app`
- Check that it shows your Plus Sandbox store (along with regular dev stores) and it allows to install and run on it

### Post-release steps

There's no need to sync, but we should also update:
- Documentation
- README files on app templates

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
